share/admin/__init__.py

```diff
@@ -249,21 +268,44 @@ class FormattedMetadataRecordAdmin(admin.ModelAdmin):
     show_full_result_count = False
 
 
-admin.site.unregister(AccessToken)
```

Comment:
Is this `unregister` irrelevant now?

Reply:
yep -- it was previously unregistered just to register the custom `AccessTokenAdmin`, but now with the customized `AdminSite`, it's not already registered